scripts/plot_p_nom_max.py

```diff
@@ -42,31 +43,17 @@ def cum_p_nom_max(net, tech, country=None):
 
 
 if __name__ == "__main__":
-    # Detect running outside of snakemake and mock snakemake for testing
     if 'snakemake' not in globals():
-        from vresutils.snakemake import MockSnakemake, Dict
-        snakemake = MockSnakemake(
-            path='..',
-            wildcards={'clusters': '45,90,181,full',
-                       'country': 'all'},
-            params=dict(techs=['onwind', 'offwind-ac', 'offwind-dc', 'solar']),
-            input=Dict(
-                **{
-                    'full': 'networks/elec_s.nc',
-                    '45': 'networks/elec_s_45.nc',
-                    '90': 'networks/elec_s_90.nc',
-                    '181': 'networks/elec_s_181.nc',
-                }
-            ),
-            output=['results/plots/cum_p_nom_max_{clusters}_{country}.pdf']
-        )
-    
+        from _helpers import mock_snakemake
+        snakemake = mock_snakemake('plot_p_nom_max', network='elec', simpl='',
+                                  technology='solar', ext='pdf', clusters= '5,full',
+                                  country= 'all')
     configure_logging(snakemake)
 
     plot_kwds = dict(drawstyle="steps-post")
 
     clusters = snakemake.wildcards.clusters.split(',')
-    techs = snakemake.params.techs
+    techs = snakemake.config['renewable'].keys()
```

Comment:
```suggestion
    techs = snakemake.wildcards.technology
```

Reply:
this seems to be conflicting with the code, which wants techs to be a list. However then the question rises why the technology wildcard is there in the first place... I see two options: 
1. keeping it as it was and delete the technology wildcard or
2. change the code to plot only one technology which is defined by the wildcard

Reply:
If you don't mind I'll keep it as it is and raise a small issue for later

Reply:
True.

I had used it w/o the technology wildcard but with a techs [param](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules), only that I forgot to add it to the `Snakefile` as I was only using it with my mocksnakemake.

I think it would be good if the snakemake.output matches with what is actually created. This is only possible with the wildcard, thus:
```suggestion
    techs = [snakemake.wildcards.tech]
```

But let's call it `tech` as we did for the `build_renewable_profile` rule. So changes for `Snakefile` are also needed.

Reply:
Ah no, due to the clusters list, this is not possible anyway :/

Reply:
yep, this makes sense

Reply:
Can you add this before merging? And maybe commit with a `[fixes #112]` tag.

Reply:
yes